examples/destination/python/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+python3 -m venv destination_run
```

Comment:
We should display some messages for the steps, otherwise it just sits there, doing a bunch of stuff, but I have no idea what's happening..

Reply:
Added comment for each step